### PR TITLE
Issue 84

### PR DESCRIPTION
**Pull request by wthompson40**
Address breakdown visualization - dynamically changes with cache parameters being changed.

### CODE REVIEW

**Comment by kevinsrose** [comment]
Nice this looks cool, is there any way to switch the placement of the breakdown visual and the delete button for the cache levels? I noticed there are certain times where the delete button gets covered up.

File: src/browser/cacheDisplay.html [hunk 1]
```diff
@@ -27,6 +27,52 @@
                         <md-button ng-click="$ctrl.removeCache(0, $event)" ng-disabled="$ctrl.cacheInfo.disableDeleteCache"><i class="material-icons">delete</i></md-button>
```
Comment (kevinsrose) [inline src/browser/cacheDisplay.html]: Ok yeah to answer my own question, it seems like moving this line below the md-card-content area you added fixes the issue brought up in my earlier comment.

**Comment by wthompson40** [comment]
Just fixed this problem, check it out now

**Review by kevinsrose — APPROVED** [review]
image

**Pull request merged**